lib/stack-functions

```diff
@@ -106,14 +121,19 @@ stack-update() {
     echo "Could not find template (${template}). You can specify alternative template as second argument."
     return 1
   fi
+
   local params=$(_stack_params_arg $inputs)
   if [ -n "$params" ]; then local parameters="--parameters file://$params"; fi
 
+  local capabilities=''
+  local caps=$(_get_stack_capabilities $stack)
+  if [ "$caps" != "" ] ; then capabilities="--capabilities $caps" ; fi
```

Comment:
`capabilities` and `caps` seem like they are the same thing... Can you make the intent read a little clearer?

Reply:
https://github.com/realestate-com-au/bash-my-aws/pull/111/commits/926fc7e8c406e63165c7db32b9e7c11446b5ea82